app/src/app/components/contents/PacketGroup/PacketTable.tsx

```diff
@@ -4,18 +4,17 @@ import { Skeleton } from "../../Base/Skeleton";
 import { DataTable } from "../common/DataTable";
 import { ErrorComponent } from "../common/ErrorComponent";
 import { Pagination } from "../common/Pagination";
-import { useGetPacketGroups } from "./hooks/useGetPacketGroups";
+import { useGetPacketPages } from "./hooks/useGetPacketPages";
```

Comment:
Another renaming: what’s returned is a paginated list of packets, not a packet group.

Reply:
But what we're showing here is a single page of packet groups, isn't it? It seems odd to send "packetPages.content" into the data table as if it's going to display multiple pages...?
Would it be better as packetPage singular?